### PR TITLE
linting fixes

### PR DESCRIPTION
**Pull request by brendankenny**
Another update for #4 

This gets `npm run lint` (and `npm test`) running without error. 90% whitespace changes, with a few tweaks to the eslint rules.

Added two TODOs for @paulirish since I'm not entirely sure what's going on there and git blame blames him :)

There are remaining eslint warnings. I'll follow up in #4

### CODE REVIEW

File: helpers/browser/driver.js [hunk 1]
```diff
@@ -61,76 +60,69 @@ class ChromeProtocol {
...
-    chrome.on("ServiceWorker.workerVersionUpdated", data => { cb(data, resolve); });
+    chrome.on("ServiceWorker.workerVersionUpdated", data => {
+      cb(data, resolve);
+    });
   }
 
-  evaluateScript(scriptStr){
-    return new Promise((function(resolve, reject) {
-
+  evaluateScript(chrome) { /* TODO(paulirish): scriptStr parameter */
```
Comment (brendankenny) [inline helpers/browser/driver.js]: @paulirish 
should `scriptStr` be passed in the `expression` property below? And this appears to need `chrome` passed in, like `profilePageLoad` below, but this doesn't appear to be called anywhere, so I wasn't sure.

File: index.js [hunk 2]
```diff
@@ -80,63 +74,63 @@ class TestRunner {
...
-            res.on('data', data => body += data);
-            res.on('end', () => {
-              try {
-                collatedOutputs.dom = DOMParser.parse(body);
-              } catch (e){
-                reject(e);
-              }
-              resolve();
-            });
+            // TODO(paulirish):
```
Comment (brendankenny) [inline index.js]: @paulirish :P

File: .eslintrc [hunk 3]
```diff
@@ -1,3 +1,22 @@
 {
-  "extends": "google"
+  "extends": "google",
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-implicit-coercion": [2, {
```
Comment (samccone) [inline .eslintrc]: wait... what is this [2 .... have the asci codes escaped into the wild

File: index.js (hunk 2 above)
Comment (samccone) [inline index.js]: :goat:

File: .eslintrc (hunk 3 above)
Comment (brendankenny) [inline .eslintrc]: ha, they really need some constants for that. 0 is off, 1 is warning, 2 is error. 2 is the default for all of these, this just adds specific exceptions

File: .eslintrc [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+      "boolean": false,
```
Comment (brendankenny) [inline .eslintrc]: defaults to `true`. Setting to false allows using `!!foo`

File: .eslintrc [hunk 5]
```diff
@@ -1,3 +1,22 @@
...
+      "number": true,
+      "string": true
+    }],
+    "no-unused-expressions": [2, {
+      "allowShortCircuit": true,
```
Comment (brendankenny) [inline .eslintrc]: defaults to `false`. Setting to true allows using `obj.fn && obj.fn()`

File: .eslintrc [hunk 6]
```diff
@@ -1,3 +1,22 @@
...
+      "allowTernary": false
+    }],
+    "no-unused-vars": [2, {
+      "vars": "all",
+      "args": "after-used",
+      "argsIgnorePattern": "(^reject$|^_$)"
```
Comment (brendankenny) [inline .eslintrc]: above this is all default, but this pattern allows `reject` in the signature of the executor passed to a Promise constructor without actually using it, and `_` as a placeholder for an unused parameter to an arrow function

File: .eslintrc (hunk 3 above)
Comment (samccone) [inline .eslintrc]: wow ok lol

**Comment by samccone** [comment]
ok going to land this we can iterate more

File: helpers/browser/driver.js (hunk 1 above)
Comment (paulirish) [inline helpers/browser/driver.js]: Yeah I'll take a swipe at this guy. I left it pretty incomplete and definitely not working.  :)

**Pull request merged**